src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -492,70 +492,6 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
     }
   }
 
-  "Campaign manager report endpoints" - {
-
-    "Create Report endpoint" - {
-
-      allHttpMethodsExcept(POST) foreach { method =>
-        s"should reject ${method.toString} method" in {
-          new RequestBuilder(method)("/trial/manager/report") ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
-            assert(!handled)
-          }
-        }
-      }
-
-      "should succeed with a create request" in {
-        Post("/trial/manager/report") ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
-          assert(status.isSuccess)
-          assertResult(OK) {status}
-        }
-      }
-
-      "non-campaign manager should fail creating request" in {
-        Post("/trial/manager/report") ~> dummyUserIdHeaders(unauthorizedUser) ~> trialApiServiceRoutes ~> check {
-          assert(status.isFailure)
-          assertResult(Forbidden) {status}
-        }
-      }
-
-    }
-
-    "Update Report endpoint" - {
-
-      allHttpMethodsExcept(PUT) foreach { method =>
-        s"should reject ${method.toString} method" in {
-          new RequestBuilder(method)("/trial/manager/report/valid") ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
-            assert(!handled)
-          }
-        }
-      }
-
-      "should succeed with an update request" in {
-        Put("/trial/manager/report/valid") ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
-          assert(status.isSuccess)
-          assertResult(OK) {status}
-        }
-      }
-
-      "invalid spreadsheet id should fail an update request" in {
-        Put("/trial/manager/report/invalid") ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
-          println(response)
-          assert(status.isFailure)
-          assertResult(NotFound) {status}
-        }
-      }
-
-      "non-campaign manager should fail an update request" in {
-        Put("/trial/manager/report/valid") ~> dummyUserIdHeaders(unauthorizedUser) ~> trialApiServiceRoutes ~> check {
-          assert(status.isFailure)
-          assertResult(Forbidden) {status}
-        }
-      }
-
-    }
-
-  }
-
```

Comment:
Since all I did was to call a pre-existing method on a schedule, I didn't add any new tests. Do you think this needs new tests?